pkgs/development/python-modules/gtts/default.nix

```diff
@@ -27,6 +27,11 @@ buildPythonPackage rec {
     hash = "sha256-ryTR7cESDO9pH5r2FBz+6JuNMEQr39hil/FSklgaIGg=";
   };
 
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail '"click >=7.1, <8.2"' '"click >=7.1, <8.3"'
```

Comment:
Use `pythonRelaxDeps`.

Reply:
That's a better idea, I'm not usually in the python space so I forgot. Thanks!